test/unit/uk/gov/hmrc/apidefinition/validators/ApiContextValidatorSpec.scala

```diff
@@ -125,44 +127,26 @@ class ApiContextValidatorSpec extends UnitSpec with MockitoSugar {
 
       val result: validatorUnderTest.HMRCValidated[String] = await(validatorUnderTest.validate(errorContext, apiDefinitionWithNewVersion)(context))
 
-      verifyValidationFailed(result,
-        Seq(s"Field 'context' must start with one of 'accounts', 'agents', 'customs', 'individuals', 'misc', 'mobile', 'obligations', 'organisations', 'payments', 'test' $errorContext",
-          s"Field 'context' must have at least two segments $errorContext"))
-    }
-
-    "fail validation when new version of existing API context is superset of existing" in new Setup {
```

Comment:
Don't we still need this ?  Even if we're not checking the "validity" of the context we should still be checking "overlaps" with other existing APIs.

Reply:
I don't think we do. There is validation in there to ensure that the context has not changed (hence the additional test in the Spec), so existing microservices are locked into those. So there shouldn't be any overlaps with existing APIs. And any new ones coming along have to meet the new requirements.

Reply:
Makes sense.  Yeah - nothing to see here.